assets/js/redirect.js

```diff
@@ -23,5 +23,43 @@
 				}
 			});
 		});
+
+		// Disable the 'Redirect To:' field unless a 4xx status code is set.
```

Comment:
Seems like this code comment says the opposite of what actually happens:

```suggestion
		// Disable the 'Redirect To:' field if a 4xx status code is set.
```